shuup/admin/modules/categories/views/list.py

```diff
@@ -17,6 +17,8 @@
 
 class CategoryListView(PicotableListView):
     model = Category
+    choices = list(CategoryStatus.choices())
+    choices.pop(2)  # Remove DELETED status
```

Comment:
`choices = [status for status in CategoryStatus if status != CategoryStatus.DELETED]` this should be better.

Reply:
Or just do simple list based on valid statuses to make things more readable since all choices are actually visible here.